src/Form/Form.js

```diff
@@ -46,11 +46,16 @@ class Form {
         }
 
         this.getValues = () => {
-            return [...this.inputs.credentials].reduce((output, input) => {
+            const credentials = [...this.inputs.credentials, ...this.inputs.emailNew].reduce((output, input) => {
                 const subtype = getInputSubtype(input)
                 output[subtype] = input.value || output[subtype]
                 return output
             }, {username: '', password: ''})
+            if (credentials.emailNew && !credentials.username) {
+                credentials.username = credentials.emailNew
+            }
```

Comment:
If we don't have a username, let's try and save the email if available.

Reply:
Can you add this as a comment :)